client/apps/edit/components/section_text/index.coffee

```diff
@@ -1,145 +1,390 @@
-#
-# Text section that display large bodies of copy and can split itself
-# between empty paragraphs to insert a new section.
-#
-
-# Using `try` here b/c Scribe is an AMD module that doesn't play nice when
-# requiring it for testing in node.
-try
-  Scribe = require 'scribe-editor'
-  scribePluginToolbar = require 'scribe-plugin-toolbar'
-  scribePluginSanitizer = require '../../lib/sanitizer.coffee'
-  scribePluginLinkTooltip = require 'scribe-plugin-enhanced-link-tooltip'
-  scribePluginKeyboardShortcuts = require 'scribe-plugin-keyboard-shortcuts'
-  scribePluginHeadingCommand = require 'scribe-plugin-heading-command'
-  scribePluginSanitizeGoogleDoc = require 'scribe-plugin-sanitize-google-doc'
-  scribePluginJumpLink = require 'scribe-plugin-jump-link'
-  scribePluginFollowLinkTooltip = require 'scribe-plugin-follow-link-tooltip'
 React = require 'react'
-icons = -> require('./icons.jade') arguments...
-{ div, nav, button } = React.DOM
+ReactDOM = require 'react-dom'
+window.global = window
 sd = require('sharify').data
+window.process = {env: {NODE_ENV: sd.NODE_ENV}}
+
+{ convertToRaw,
+  CompositeDecorator,
+  ContentState,
+  Editor,
+  EditorState,
+  Entity,
+  RichUtils,
+  Modifier,
+  DefaultDraftBlockRenderMap,
+  getVisibleSelectionRect } = require 'draft-js'
+{ convertFromHTML, convertToHTML } = require 'draft-convert'
+Immutable = require 'immutable'
+Decorators = require '../../../../components/rich_text/decorators.coffee'
+icons = -> require('../../../../components/rich_text/icons.jade') arguments...
+{ div, nav, a, button, span, p, br } = React.DOM
+ButtonStyle = React.createFactory require '../../../../components/rich_text/components/button_style.coffee'
+InputUrl = React.createFactory require '../../../../components/rich_text/components/input_url.coffee'
+editor = (props) -> React.createElement Editor, props
 Channel = require '../../../../models/channel.coffee'
 
-keyboardShortcutsMap =
-  bold: (e) -> e.metaKey and e.keyCode is 66
-  italic: (e) -> e.metaKey and e.keyCode is 73
-  h2: (e) -> e.metaKey && e.keyCode is 50
-  h3: (e) -> e.metaKey && e.keyCode is 51
-  removeFormat: (e) -> e.altKey and e.shiftKey and e.keyCode is 65
-  linkPrompt: (e) -> e.metaKey and not e.shiftKey and e.keyCode is 75
-  unlink: (e) -> e.metaKey and e.shiftKey and e.keyCode is 75
-  insertOrderedList: (e) -> e.metaKey and e.shiftKey and e.keyCode is 56
-  insertUnorderedList: (e) -> e.metaKey and e.shiftKey and e.keyCode is 55
```

Comment:
Have been speaking with @halleyjohnson about this, need to do a little more research on overriding browser/os commands, as I haven't yet gotten their key command override functioning correctly for multiple keys at once. We currently have clipboard events, undo/redo, bold and italic.